kevintdd/src/main/java/Store.java

```diff
@@ -0,0 +1,62 @@
+import java.math.BigDecimal;
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals
[Store.txt](https://github.com/KTBG7/tdd/files/4956435/Store.txt)